documentation/staging/content/samples/azure-kubernetes-service/_index.md

```diff
@@ -20,6 +20,10 @@ description: "Sample for using the operator to set up a WLS cluster on the Azure
 
 This sample demonstrates how to use the [WebLogic Kubernetes Operator]({{< relref "/_index.md" >}}) (hereafter "the operator") to set up a WebLogic Server (WLS) cluster on the Azure Kubernetes Service (AKS). After going through the steps, your WLS domain runs on an AKS cluster.  You have several options for managing the cluster, depending on which [domain home source type]({{< relref "/userguide/managing-domains/choosing-a-model/_index.md" >}}) you choose.  With Domain in PV, you can manage your WLS domain by accessing the WebLogic Server Administration Console or WLST.  With Model in Image, you use the operator to perform WLS administrative operations.
 
+**Note:** For an alternative approach to this sample, see the [WebLogic Server on AKS Marketplace]({{<relref "/userguide/aks/_index.md">}}) offering which automates the provisioning the provisioning of
```

Comment:
automates the provisioning the provisioning of -> automates the provisioning of

Reply:
fixed

Reply:
Resolved.